app/models/user.rb

```diff
@@ -89,4 +92,16 @@ def default_email_record
     result ||= valid_emails.first
     result
   end
+
+  # Overrides `Devise::Models::Validatable`
```

Comment:
But overriding these methods are for the validation of system user, so I think it belongs to here. And concern will cross reference the system user here, which is coupled ?  what do you think ?